internal/core/thirdparty/simdjson/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@
 
 FetchContent_Declare(
     simdjson
-    GIT_REPOSITORY https://github.com/simdjson/simdjson.git
-    GIT_TAG        v3.1.7
+    URL https://github.com/simdjson/simdjson/archive/refs/tags/v3.1.7.tar.gz
```

Comment:
Please added URL_HASH MD5=1b0d75ad32179c77f84f4a09d4214057

or it'll try to download everytime.

Reply:
ref: https://cmake.org/cmake/help/latest/module/FetchContent.html
![image](https://github.com/milvus-io/milvus/assets/51370125/054239e8-9023-48f5-8dc5-ca8573e934ca)